src/components/ListUniqueValues.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <div class="list-unique-values">
+    <div class="list-unique-values__title">Filter values</div>
+    <input class="list-unique-values__search-box" placeholder="Search 50 items" v-model="search" />
+    <div class="list-unique-values__select-clear-all">
+      <span class="list-unique-values__select-all" @click="selectAll">Select all</span> &nbsp;
+      <span class="list-unique-values__clear-all" @click="clearAll">Clear all</span>
+    </div>
+    <div
+      class="list-unique-values__checkbox-container"
+      v-for="option in searchedOptions"
+      :key="option.value"
+    >
+      <CheckboxWidget
+        :label="`${option.value} (${option.count})`"
+        :value="isChecked(option)"
+        @input="toggleCheck(option)"
+      />
+    </div>
+    <div class="list-unique-values__load-all-values" v-if="loaded">
+      <div>List maybe incomplete</div>
+      <div @click="loadAllValues" class="list-unique-values__load-all-values-button">
+        Load all values
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import _difference from 'lodash/difference';
+import _union from 'lodash/union';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+import { FilterConditionInclusion } from '@/lib/steps.ts';
+
+// This type will be imported from @/lib/dataset/helpers.ts
+type ColumnValueStat = {
+  value: any;
+  count: number;
+};
+
+import CheckboxWidget from './stepforms/widgets/Checkbox.vue';
+@Component({
+  name: 'list-unique-values',
+  components: { CheckboxWidget },
+})
+export default class ListUniqueValues extends Vue {
+  @Prop({
+    type: Array,
+    required: true,
+  })
+  options!: ColumnValueStat[];
+
+  @Prop({
+    type: Object,
+    required: true,
+  })
+  filter!: FilterConditionInclusion;
```

Comment:
I think there is no need for this component to know which column it is editing.
You can declare the sub type like this in steps.ts:
```
export type ConditionInclusion = {
  value: any[];
  operator: 'in' | 'nin';
};

FilterConditionInclusion = ConditionInclusion & {
  column: string;
};

Reply:
Summary of Slack discussion:
@rhuille prefer avoiding type multiplication, and keep a string link between this and the filter step, and the ability in the short future to turn this object into a real filter step condition
I still advise against keeping the `column` attribute, but it's not an important point so let's go with this :)